docs/quickstarts/javascript.mdx

```diff
@@ -177,6 +177,8 @@ Now that you have an application integrated with Clerk, you will want to read th
 
     <Cards title="Authentication Components" description="Learn more about all our authentication components." link="/docs/references/javascript/clerk/clerk#components" cta="Learn More" />
 
+    <Cards title="React Hooks" description="Leverage Clerk React hooks for your Next.js application." link="/docs/references/react/use-user" cta="Learn More" />
+
```

Comment:
Let's remove this one, I don't think it makes sense in the context of the JS quickstart, which doesn't require the use of React or Next.

```suggestion
```

Reply:
I definitely added that to the wrong place!!